src/google/protobuf/stubs/common.h

```diff
@@ -47,6 +47,17 @@
 #include "google/protobuf/stubs/platform_macros.h"
 #include "google/protobuf/stubs/port.h"
 
+// Enforce C++14 as the minimum.
+#if defined(_MSVC_LANG)
+#if _MSVC_LANG < 201402L
+#error "C++ versions less than C++14 are not supported."
+#endif  // _MSVC_LANG < 201402L
+#elif defined(__cplusplus)
```

Comment:
Could you just special-case gcc < 5.0 here?  We don't support it anyway so it should be safe to add, and will unblock the aarch64 tests.  e.g.
#elif defined(__cplusplus) || (defined(__GNUC__) && __GNUC__ < 5)

Reply:
It looks like this version of gcc sets \_\_cplusplus to 201300L

Reply:
okay i set a special case for this; this will have to be backported to the guard in absl for these tests to pass there if we update the absl pin

Reply:
Ah thanks for the heads up!  I'll try to prioritize this issue then, because we definitely want to bump our Abseil version before the next release